docs/develop/smart-contracts/outposts/outposts.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 1
+---
+
+# Outposts
+
+Outposts leverage the
+[IBC Transfer Extension](https://docs.evmos.org/develop/smart-contracts/evm-extensions/ibc-transfer)
+to interact with other chains through the creation of a custom `memo` field that holds instructions for the
+reception chain. This allows users to interact with other chains without leaving the Evmos chain. Evmos supports
+two outposts at the moment:
+
+1. Osmosis Cross Chain Swaps Outpost
+2. Stride Liquid Staking and Redeeming Outpost
+
+## Key Components
+
+### Common
+
+- **`memo` field**: An extra JSON string attached to IBC Transfer packets. This additional information helps
+  the receiving chain to identify the user and action to be performed.
+- **IBC Extension**: A precompiled contract that allows users to send IBC transfers to other chains through the EVM.
+- **ICS20 Transfers**: Protocol for transferring fungible tokens using Inter-Blockchain Communication (IBC).
```

Comment:
nit: Could be more precise, since this is specifically the `ICS20 Extension`, we will probably have some other IBC related precompiles in the future 👀 
```suggestion
- **ICS20 Transfers**: Protocol for transferring fungible tokens using Inter-Blockchain Communication (IBC).
- **ICS20 Extension**: A precompiled contract that allows users to send IBC transfers to other chains through the EVM.
```